source/compilers.md

```diff
@@ -36,6 +36,8 @@ based on the NVIDIA/PGI commercial compiler.
 [LFortran](https://lfortran.org) is a modern, interactive, LLVM-based Fortran
 compiler.
 
+[LFortran WASM Compiler](https://dev.lfortran.org/) is a statically served, WASM based LFortran Compiler for running the code in the web browsers.
+
```

Comment:
How about connecting it with the previous item, something like this:
```suggestion
Also available online at [dev.lfortran.org](https://dev.lfortran.org/) using a statically served, WASM based LFortran Compiler for running code in web browsers.
```

Reply:
Thanks sir.